doc/plutus/explanations/plutus-tools-component-descriptions.rst

```diff
@@ -0,0 +1,241 @@
+Once you have defined a suitable model, then QuickCheck can generate and run many thousands of scenarios, taking the application through a wide variety of states, and checking that it behaves correctly in each one. 
+
+   See also: 
+   
+   * :doc:`Property-based testing of Plutus contracts <../tutorials/contract-testing>` 
+   * :doc:`Testing Plutus Contracts with Contract Models <../tutorials/contract-models>` 
+
+3. Contract monad emulator
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++--------------+---------------------------------------------------------------+
+| Contract monad emulator                                                      |
++==============+===============================================================+
+| Location     | ``plutus-apps/plutus-contract``                               |
++--------------+---------------------------------------------------------------+
+
+`Contract monad emulator <https://github.com/input-output-hk/plutus-apps/tree/main/plutus-contract>`_ is a library that provides an environment for emulating the blockchain. 
+The environment provides a way for writing traces for the contract which are sequences of actions by simulated wallets that use the contract. 
+The component is highly dependent on the Contract API (Contract monad). 
+
+4. Contract API (also known as Contract monad)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++--------------+---------------------------------------------------------------+
+| Contract API                                                                 |
++==============+===============================================================+
+| Location     | ``plutus-apps/plutus-contract``                               |
++--------------+---------------------------------------------------------------+
+
+`Contract API <https://github.com/input-output-hk/plutus-apps/tree/main/plutus-contract>`_ is a logical component within the Plutus Contract package, providing an effective system for describing smart contracts that interact with wallets, DApps and the blockchain. 
+It provides the Contract API interface for writing the off-chain part of a Plutus application that is to be interpreted by an emulator or by Plutus application backend (PAB). 
+
+5. Plutus chain index
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++--------------+---------------------------------------------------------------+
+| Plutus chain index                                                           |
++==============+===============================================================+
+| Location     | ``plutus-apps/plutus-chain-index-core``                       |
+|              | ``plutus-apps/plutus-chain-index``                            |
++--------------+---------------------------------------------------------------+
+
+   COMMENT: Ziyang suggests that we remove this one from the page. Does anyone have other opinions? 
```

Comment:
There is no replacement for `plutus-chain-index` right now. I would like to leave it here until we get one.

Reply:
Agree

Reply:
Noted.